File: PARAGENESIS/test.fns/f6.cs
double:physical Peval(str, length, vect, genes)
char:physical str[];
int length;
double:physical vect[];
int genes;
{
  register int i;
  double:physical sum;

  with(physical) {
    sum = 0.0;
    for (i = 0; i < length; i++)
      sum += (str[i] == '0');
    
    return (sum);
  }
}

File: genesis/test.fns/f6.c
/************************************************  file f6.c  ****/

double eval(str, length, vect, genes)
char str[];	/* string representation			*/
int length;	/* length of bit string				*/
double vect[];	/* floating point representation		*/
int genes;	/* number of elements in vect			*/
{
	register int i;
	double sum;

	sum = 0.0;
	for (i = 0; i < length; i++)
		sum += (str[i] == '0');

	return (sum);
}

/************************************************ end of file ****/
